util/src/lib.rs

```diff
@@ -133,26 +135,35 @@ pub async fn get_metadata_sections(
     let comments = api
         .all_pages(api_issues.list_comments(pull_nr).send().await?)
         .await?;
+
+    Ok(get_metadata_sections_from_comments(&comments, pull_nr))
+}
+
+pub fn get_metadata_sections_from_comments(
+    comments: &Vec<octocrab::models::issues::Comment>,
+    pull_nr: u64,
+) -> MetaComment {
     for c in comments {
-        let b = c.body.expect("remote api error");
+        let b = c.body.clone().expect("remote api error");
```

Comment:
any reason to create a copy when a reference is enough?
```suggestion
        let b = c.body.as_ref().expect("remote api error");
```